lms/static/scripts/frontend_apps/components/test/FilePickerApp-test.js

```diff
@@ -506,6 +508,91 @@ describe('FilePickerApp', () => {
     assert.isFalse(wrapper.exists('ErrorModal'));
   });
 
+  context('when promptForGradable is enabled', () => {
+    const deepLinkingAPIPath = '/lti/1.3/deep_linking/form_fields';
+    const deepLinkingAPIData = { some: 'data' };
+    let fakeAPICall;
+    let fakeFormFields;
+
+    beforeEach(() => {
+      fakeConfig.filePicker.deepLinkingAPI = {
+        path: deepLinkingAPIPath,
+        data: deepLinkingAPIData,
+      };
+      fakeConfig.filePicker.promptForGradable = true;
+
+      fakeAPICall = sinon.stub();
+      fakeFormFields = { JWT: 'JWT VALUE' };
+
+      fakeAPICall
+        .withArgs(sinon.match({ path: deepLinkingAPIPath }))
+        .resolves(fakeFormFields);
+
+      $imports.$mock({
+        '../utils/api': { apiCall: fakeAPICall },
+      });
+    });
+
+    it('displays "max points" input on details screen', () => {
+      const wrapper = renderFilePicker();
+      selectContent(wrapper, 'https://example.com');
+      const gradableCheckbox = wrapper.find(
+        'input[data-testid="gradable-max-input"]',
+      );
+      assert.isTrue(gradableCheckbox.exists());
+    });
+
+    it('includes max points when submitting form', async () => {
+      const onSubmit = sinon.stub().callsFake(e => e.preventDefault());
+      const wrapper = renderFilePicker({ onSubmit });
+      selectContent(wrapper, 'https://example.com');
+
+      const pointsInput = wrapper.find(
+        'input[data-testid="gradable-max-input"]',
+      );
+      pointsInput.getDOMNode().value = '10';
+      pointsInput.simulate('change');
+
+      interact(wrapper, () => {
+        wrapper.find('Button[data-testid="save-button"]').props().onClick();
+      });
+
+      await waitFor(() => fakeAPICall.called);
+      assert.calledWith(fakeAPICall, {
+        authToken: 'DUMMY_AUTH_TOKEN',
+        path: deepLinkingAPIPath,
+        data: {
+          ...deepLinkingAPIData,
+          content: { type: 'url', url: 'https://example.com' },
+          title: null,
+          group_set: null,
+          auto_grading_config: null,
+          assignment_gradable_max_points: 10,
+        },
+      });
+
+      await waitFor(() => onSubmit.called);
```

Comment:
There's a bit of duplicated code here that maybe it's worth extracting to a local utility function.

We don't need to do that as part of this PR. Just adding this comment as a future note for any of us.